components/script/dom/bluetooth.rs

```diff
@@ -40,12 +56,155 @@ impl Bluetooth {
     }
 }
 
+impl Clone for StringOrUnsignedLong {
+    fn clone(&self) -> StringOrUnsignedLong {
+        match self {
+            &StringOrUnsignedLong::String(ref s) => StringOrUnsignedLong::String(s.clone()),
```

Comment:
Ezt ki kellene venni ha a Valentin patch-e megy be előbb vagy fordítva, az övéből kiszedni